nikola/utils.py

```diff
@@ -946,6 +946,8 @@ def extract_all(zipfile, path='themes'):
 def to_datetime(value, tzinfo=None):
     """Convert string to datetime."""
     try:
+        if isinstance(value, datetime.date):
+            value = datetime.datetime.combine(value, datetime.datetime.min.time())
```

Comment:
Are we guaranteed that datetime.datetime.min.time() is midnight?  I’d prefer to spell out midnight directly instead of going through that object.

Reply:
Yep, done.